tests/clients/web/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
 # All of our core javascript will be checked with a single test
-add_javascript_style_test(core src)
```

Comment:
Since we are keeping the function `add_javascript_style_test` around, would it make sense to style execute it  (that way it would be tested) ? and if executed outside of girder core ... it would display a CMake author warning. ?